### ISSUE TITLE
Random disconnects with error "SERVER_TERMINATED_DISPLAY_TOPOLOGY_CHANGED"

### ISSUE THREAD

**Issue by SoyG** [moonlight-stream/moonlight-qt#1062]
Hello! I'm hoping I can get some assistance with this issue. I'm getting disconnected at random intervals without any error message on the client. It happens whether a game is running or just at the desktop, and at any resolution/fps setting. The issue happens on my local network or when using the Internet hosting tool. When the client is disconnected, it acts like I manually did the "ctrl alt shift q" command to quit, and there's no error message or network connection warning.

I have Windows 11 on both host and client, with "GPU Scheduling" disabled. This issue happens on 2 different video cards, neither is overclocked. Host is currently running nvidia 3080, but previously was on 2080 Ti with the same issue.

I've tried running it at 1080p and 720, fullscreen and windowed, 30 and 60 fps. I have forced software decoding, vsync and frame pacing on and off, and bitrates from 10 to 20 Mbps. None of these settings has any affect on the server disconnecting. 

I'm including the log file, but here are a couple lines that caught my eye. I'm hoping someone can read it and locate the actual reason for the server quitting:

```
#1(I)[2023-08-30 20:35:30,582]=16:35:30={39568}<ResultUtils>      Termination reason = SERVER_TERMINATED_DISPLAY_TOPOLOGY_CHANGED (SRC_DisplayTopologyChanged)
#2(I)[2023-08-30 20:35:30,582]=16:35:30={11560}<Server>           Server has been asked to stop.
#3(I)[2023-08-30 20:35:30,582]=16:35:30={11560}<Server>           Server has been asked to end the current streaming session.
#4(I)[2023-08-30 20:35:30,582]=16:35:30={11560}<Server>           Server is now in state: EndingSession
#5(D)[2023-08-30 20:35:30,582]=16:35:30={11560}<StopEvent>        Setting event 'Session'. Server:  (0x80030023 NVST_DISCONN_SERVER_TERMINATED_CLOSED)
```
Also this message makes me suspect an audio issue possibly? 

```
#8(I)[2023-08-30 20:35:30,582]=16:35:30={44184}<StreamSdkEvents>  Invoking diagnosticsEventRaised TerminationReason
#9(I)[2023-08-30 20:35:30,582]=16:35:30={44184}<NvStreamServerCo> DiagnosticEvent: Reported UnifiedErrorCode 430100 for SessionTerminationReason 100
#0(E)[2023-08-30 20:35:30,584]=16:35:30={19056}<NvAudCapAudioSou> default device changed, reinitializing NvVAD
#1(E)[2023-08-30 20:35:30,591]=16:35:30={39516}<VirtualVsyncServ> nvstVirtualVsyncSetActualStreamingIntervals failed due to StreamSdkException 800b1000 [NVST_R_INVALID_STATE] Cannot run the requested logic for state (1005) is incorrect.
```
[NvStreamerOld.log](https://github.com/moonlight-stream/moonlight-qt/files/12480176/NvStreamerOld.log)

**Comment by postcores-user1** [comment]
Disclosure, I'm not a member of the Moonlight support team. Are you still having this issue?

**Comment by SoyG** [comment]
> Disclosure, I'm not a member of the Moonlight support team. Are you still having this issue?

Yes. I have not found any additional information, other than it doesn't seem to affect Steam Link or Steam Remote Play as frequently
